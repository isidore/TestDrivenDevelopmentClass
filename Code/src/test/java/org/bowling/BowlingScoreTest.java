package org.bowling;

import org.approvaltests.Approvals;
import org.junit.Test;

public class BowlingScoreTest {

	@Test
	public void testOwensGame() {
		BowlingGame game = new BowlingGame();
		game.roll(0, 7, 8, 1, 10, 5, 5, 10, 8, 1, 10, 9, 0, 9, 1, 10, 10, 10);
		Approvals.verify(game);
	}

}
