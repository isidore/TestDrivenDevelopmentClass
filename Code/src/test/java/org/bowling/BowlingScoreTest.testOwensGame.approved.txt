 1 |  0  7  | 7 | 7 |
 2 |  8  1  | 9 | 16 |
